DLCQuestipelagoInstaller/Program.cs

```diff
@@ -166,7 +166,15 @@ private static string FindDLCQuest()
             }
 
             var steamInstallPath = (string)steamInstallRegistryNode;
-            var dlcQuestInstall = Path.Combine(steamInstallPath, "steamapps", "common", "DLC Quest");
+            var libraryFolders = from line in File.ReadAllLines(Path.Combine(steamInstallPath, "steamapps", "libraryfolders.vdf"))

```

Comment:
Could this Linq be written in method syntax, instead of query syntax, for consistency with the rest of the code base?

Reply:
It can and it now has been :)